### ISSUE TITLE
Installing Requirements From Dreambooth Extension Breaks Colab Notebook

### ISSUE THREAD

**Issue by theovercomer8** [TheLastBen/fast-stable-diffusion#1398]
### Discussed in https://github.com/TheLastBen/fast-stable-diffusion/discussions/1397

<div type='discussions-op-text'>

<sup>Originally posted by **theovercomer8** January 16, 2023</sup>
When I try to use the WebUI notebook, I have to pip install the dreambooth extension requirements file in order to use the extension. However, when I do so it causes the following warning upon starting the UI:
`2023-01-16 16:41:19.083681: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-01-16 16:41:21.396197: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/lib/python3.8/dist-packages/cv2/../../lib64:/usr/lib64-nvidia
2023-01-16 16:41:21.396391: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/lib/python3.8/dist-packages/cv2/../../lib64:/usr/lib64-nvidia
2023-01-16 16:41:21.396416: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.`

Then when I try to generate a 2.1 image it throws errors:
`RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper__index_select)`</div>

**Comment by TheLastBen** [comment]
because it forces reinstalling too many dependencies : https://github.com/d8ahazard/sd_dreambooth_extension/blob/main/requirements.txt

but if you disconnect and reconnect, the UI will work again.

**Comment by mediocreatmybest** [comment]
> ### Discussed in #1397
> Originally posted by **theovercomer8** January 16, 2023 When I try to use the WebUI notebook, I have to pip install the dreambooth extension requirements file in order to use the extension. However, when I do so it causes the following warning upon starting the UI: `2023-01-16 16:41:19.083681: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations: AVX2 FMA To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags. 2023-01-16 16:41:21.396197: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/lib/python3.8/dist-packages/cv2/../../lib64:/usr/lib64-nvidia 2023-01-16 16:41:21.396391: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/lib/python3.8/dist-packages/cv2/../../lib64:/usr/lib64-nvidia 2023-01-16 16:41:21.396416: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.`
> 
> Then when I try to generate a 2.1 image it throws errors: `RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument index in method wrapper__index_select)`

Yep, installing all of the requirements from the extension pretty much kills it as it also forces Gradio to be updated. 

I've successfully gotten it working and training a model by removing the dependencies section and reinstalling with this requirement file I made for testing. 
https://github.com/mediocreatmybest/gaslightingeveryone/blob/main/Colab/files/test_requirements.txt
Just a warning, it takes about 5 or so minutes just to install the dependencies. 

The colab I used for testing was slightly modified, I tried to make as little changes as possible to get it working. 
Zero support and it may or may not work for you! As usual, your results may vary and check the code before running it. 

https://github.com/mediocreatmybest/gaslightingeveryone/blob/main/Colab/Slow_Stable_Diffusion_AUTOMATIC1111.ipynb
